client/src/components/ProfileSettings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,26 @@
+import { Avatar, Box, Button, Paper } from '@material-ui/core';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+import Typography from '@material-ui/core/Typography';
```

Comment:
You can remove this import here and add it as part of the import on Line 1 :)

Reply:
Sounds good! Will update and merge